SPECS/mariner-release/mariner-release.spec

```diff
@@ -1,38 +1,37 @@
-Summary:       CBL-Mariner release files
-Name:          mariner-release
-Version:       1.0
-Release:       16%{?dist}
-License:       MIT
-Group:         System Environment/Base
-URL:           https://aka.ms/cbl-mariner
-Vendor:        Microsoft Corporation
-Distribution:  Mariner
-BuildArch:     noarch
-
+Summary:        CBL-Mariner release files
+Name:           mariner-release
+Version:        2.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://aka.ms/cbl-mariner
 # Allows package management tools to find and set the default value
 # for the "releasever" variable from the RPM database.
-Provides: system-release(releasever)
+Provides:       system-release(releasever)
+BuildArch:      noarch
 
 %description
-Azure CBL-Mariner release files such as yum configs and other /etc/ release related files
+Azure CBL-Mariner release files such as yum configs and other %{_sysconfdir}/ release related files
 
 %install
-rm -rf $RPM_BUILD_ROOT
-install -d $RPM_BUILD_ROOT/etc
-install -d $RPM_BUILD_ROOT/usr/lib
+rm -rf %{buildroot}
+install -d %{buildroot}%{_sysconfdir}
+install -d %{buildroot}%{_lib}
```

Comment:
Watch out for `%{_lib}`! I think our linter is misconfigured here and we should be using `%{_libdir}` instead. From our `SPECS/mariner-rpm-macros/macros` file:

```
%_lib               lib
(...)
%_libdir            /usr/%{_lib}
```

@thcrain-msft: where do we keep the linter's configuration? I forgot and I'd like to adjust this, while I still remember.:)

Reply:
I did look into this and I see that the /lib folder on CBL-Mariner is actually a link to /usr/lib....so they appear to be synonymous.  I reasoned that since /lib is supposed to be system installed libraries and /usr/lib is supposed to be user installed libraries, that /lib was correct for the mariner-release stuff.  If lib and /usr/lib are ever separated I think the right thing will happen.

Reply:
Got it, it should work in this case but you watch out for one more thing: I believe `%{buildroot}` doesn't end with a `/", and `%{lib}` doesn't start with one, so the path might be incorrect and we'd have to at least add that separator between them. I'm not 100% sure about `%{buildroot}`, however.

Also, using `%{lib}` might not always work if `make` or other tool installs the files under `/usr/lib`, thus I think we should still update the linter, but that's out of scope for this PR.

Reply:
Looks to me like %{buildroot} is used in quite a few spec files without a trailing /.